app/components/signup-email-input.js

```diff
@@ -2,7 +2,14 @@ import Component from '@ember/component';
 import { not, empty, and, alias } from '@ember/object/computed';
 import { inject as service } from '@ember/service';
 import { once, debounce, cancel } from '@ember/runloop';
-import { set, observer, get, computed } from '@ember/object';
+import { set, get, computed } from '@ember/object';
+import Ember from 'ember';
+
+const {
+  testing
+} = Ember;
+
```

Comment:
One thought is to make this a variable in the `environment.js` file.  DEBOUNCE_EMAIL_TIMER = 0 in `testing` and = 500 in all other cases.  Then just import `DEBOUNCE_EMAIL_TIMER`

Reply:
Sounds like a good thing to add later for sure. For now, I'm not waiting on another 7 mins to build this 😛